tripy/tripy/function_registry.py

```diff
@@ -102,14 +104,23 @@ def matches_type(name: str, annotation: type, arg: Any):
             # In cases where a type is not available at the time of function definition, the type
             # annotation may be provided as a string. Since we need the actual type, we just
             # eval it here.
-            if isinstance(annotation, str):
+            import importlib
+
+            if isinstance(annotation, (str, ForwardRef)):
```

Comment:
@slyubomirsky also implemented this in his PR: https://github.com/NVIDIA/TensorRT-Incubator/pull/185
Which one should we keep?

Reply:
I can use @slyubomirsky's implementation since it implements broader support.

Reply:
Just curious, where did `ForwardRef` come into play here? I did not know about `__forward_arg__`, so I think I could use this to handle tensor literals without a workaround.